README.md

```diff
@@ -1,107 +1,120 @@
 
 An example bash session under linux is shown below:
 
 ```bash
-cfw@dm-wrkstn:/tmp$ git clone git@github.com:JiscRDSS/rdss-par.git
-Cloning into 'rdss-par'...
-remote: Enumerating objects: 661, done.
-remote: Total 661 (delta 0), reused 0 (delta 0), pack-reused 661
-Receiving objects: 100% (661/661), 1.53 MiB | 1.54 MiB/s, done.
-Resolving deltas: 100% (358/358), done.
-
-cfw@dm-wrkstn:/tmp$ cd rdss-par/
-
-cfw@dm-wrkstn:/tmp/rdss-par$ git submodule update --init --recursive
-Submodule 'examples' (https://github.com/JiscRDSS/rdss-par-examples.git) registered for path 'examples'
-Cloning into '/tmp/rdss-par/examples'...
-Submodule path 'examples': checked out 'edec3997b38e73d967e7ba1ba3c76345e02097f0'
-
-cfw@dm-wrkstn:/tmp/rdss-par$
+❯ git clone https://github.com/openpreserve/par.git
+Cloning into 'par'...
+remote: Enumerating objects: 1810, done.
+remote: Counting objects: 100% (103/103), done.
+remote: Compressing objects: 100% (68/68), done.
+remote: Total 1810 (delta 34), reused 83 (delta 29), pack-reused 1707
+Receiving objects: 100% (1810/1810), 9.12 MiB | 27.00 KiB/s, done.
+Resolving deltas: 100% (939/939), done.
+❯ cd par
+❯ git submodule update --init --recursive
+Submodule 'examples' (https://github.com/openpreserve/par-examples.git) registered for path 'examples'
+Cloning into '/tmp/par/examples'...
+Submodule path 'examples': checked out '0ea43f0a0484e02615515186d9e976d5f0ddf8ca'
 ```
 
 ### Project Layout
-```
-rdss-par
-    |- api          The PAR API definitions and documentation.
-    |- docs         The contents of the GitHub pages parecore.org site.
-    |- examples     Practical JSON examples of data model usage.
-    |- schemas      The PAR data model defined as JSON schema.
-    |- scripts      Helper scripts for testing, etc.
-    |- testing      Pytests that validate the examples using the schema.
+
+```shell
+par
+ |- api          The PAR API definitions and documentation.
+ |- docs         The contents of the GitHub pages parecore.org site.
+ |- examples     Practical JSON examples of data model usage.
+ |- schema       The PAR data model defined as JSON schema.
+ |- scripts      Helper scripts for testing, etc.
+ |- testing      Pytests that validate the examples using the schema.
 ```
 
 ### Dependencies
-To run the tests you'll need [Python](https://www.python.org/) 2.7 or higher with the following:
+
+To run the tests you'll need [Python](https://www.python.org/) 3.8 or higher with the following:
+
 - [`pip`](https://pypi.org/project/pip/) to install Python dependencies;
 - [`demjson`](https://pypi.org/project/demjson/) for command line JSON linting;
 - [`jsonschema`](https://pypi.org/project/jsonschema/) for JSON schema validation;
 - [`pytest`](https://pypi.org/project/pytest/) to run the schema validation tests.
 
 Once you have Pythton and pip you can install the other dependencies so: `pip install -r requirements.txt`
 
+If the installation of demjson fails with an error similar to the below:
+
```

Comment:
assuming you were intending to document an error case?

Reply:
Hah, good catch but should have left that out.